test/instrumentation/modules/@elastic/elasticsearch.test.js

```diff
@@ -542,17 +541,11 @@ ${body.map(JSON.stringify).join('\n')}
 
           const err = data.errors
             .filter((e) => e.exception.type === 'RequestAbortedError')[0]
-          if (semver.satisfies(esVersion, '7.14.x')) {
-            // https://github.com/elastic/elasticsearch-js/issues/1517 was fixed
```

Comment:
REVIEW NOTE: This issue was only relevant when using the diagnostics events from the ES client. Given that the instrumentation re-write is no longer using those events, we no longer need to have the tests workaround the issue.